src/Enzo/enzo_EnzoMethodHydro.cpp

```diff
@@ -1,10 +1,16 @@
+//
+// This is not used, but is a starting point to refactor current
```

Comment:
Great, I'm really happy to see this. I found the `EnzoMethodHydro` method to be really confusing when I was first getting acquainted with the codebase.